test/parallel/test-module-isBuiltIn.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const { isBuiltIn } = require('module');
+
+// Includes modules in lib/ (even deprecated ones)
+assert(isBuiltIn('http'));
+assert(isBuiltIn('sys'));
```

Comment:
```suggestion
assert(isBuiltIn('sys'));
assert(isBuiltIn('node:test'));
```

Reply:
What's the diff here? 🤔

Reply:
To make sure it loads with prefixed modules and modules that are only available via prefix?

Reply:
Ha, I had added them in the doc and not in the tests, thanks.